docs/changelog.md

```diff
@@ -1,5 +1,49 @@
 ## Changelog
 
+### v0.13.0 (1/5/2021)
+
+#### Highlights
+- Support a monocular 3D detection method [FCOS3D](https://arxiv.org/abs/2104.10956)
+- Support ScanNet and S3DIS semantic segmentation dataset
+- Enhancement of visualization tools for datasets browsing, multi-modality and more tasks
```

Comment:
for dataset browsing and demos, including support of visualization of multi-modality data and LiDAR segmentation data.